### PR TITLE
Promote AdmissionWebHook should be able to deny pod and configmap creation to Conformance

### LINKED ISSUE #78832: Promote ConfigMap Tests to Conformance
Using the list of tested stable endpoints referenced in https://github.com/kubernetes/kubernetes/issues/78748, we can pick out the endpoints which are part of the ConfigMap kind:

[patchCoreV1NamespacedConfigMap](https://apisnoop.cncf.io/?zoomed=operationId-stable-core-patchCoreV1NamespacedConfigMap&showUntested=false&showConformanceTested=false)

This is a stable core endpoint which has existing testing, potentially making for low-hanging fruit to upgrade to Conformance.

/area test
/area conformance
/sig testing

### PR DESCRIPTION
**Pull request by hh**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

Promotes the following E2E test to Conformance:
```test/e2e/apimachinery/webhook.go: "Should be able to deny pod and configmap creation"```

**Which issue(s) this PR fixes**:

Fixes: #78832

**Special notes for your reviewer**:

Part of Umbrella Issue #78748

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

/area conformance
/area test

@kubernetes/sig-architecture-pr-reviews
@kubernetes/cncf-conformance-wg
@kubernetes/sig-api-machinery-pr-reviews

### CODE REVIEW

File: test/e2e/apimachinery/webhook.go [hunk 1]
```diff
@@ -125,7 +125,12 @@ var _ = SIGDescribe("AdmissionWebhook", func() {
 		cleanWebhookTest(client, namespaceName)
 	})
 
-	ginkgo.It("Should be able to deny pod and configmap creation", func() {
+	/*
+		Release : v1.16
+		Testname: LimitRange: CRUD
```
Comment (spiffxp) [inline test/e2e/apimachinery/webhook.go]: copy paste

**Comment by roycaihw** [comment]
nits, just fyi: 

- This test hits the [patchCoreV1NamespacedConfigMap](https://github.com/kubernetes/kubernetes/issues/78832#issuecomment-502925334) endpoint, but it expects the admission plugin to [reject the request](https://github.com/kubernetes/kubernetes/blob/c898ab2043655758f07c412a98c7388e5b4dea69/test/e2e/apimachinery/webhook.go#L761-L767)
- According to [conformance test requirements](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md#conformance-test-requirements), a conformance test uses only GA APIs. Currently AdmissionWebhook is still beta but plan to graduate to GA in 1.16

File: test/e2e/apimachinery/webhook.go (hunk 1 above)
Comment (timothysc) [reply test/e2e/apimachinery/webhook.go]: There is also a very long todo at the end of this also long test... 

// TODO: add more e2e tests for mutating webhooks
	// 1. mutating webhook that mutates pod		// 1. mutating webhook that mutates pod
	// 2. mutating webhook that sends empty patch		// 2. mutating webhook that sends empty patch
	//   2.1 and sets status.allowed=true		//   2.1 and sets status.allowed=true
	//   2.2 and sets status.allowed=false		//   2.2 and sets status.allowed=false
	// 3. mutating webhook that sends patch, but also sets status.allowed=false		// 3. mutating webhook that sends patch, but also sets status.allowed=false
	// 4. mutating webhook that fail-open v.s. fail-closed		// 4. mutating webhook that fail-open v.s. fail-closed

Comment (liggitt) [reply test/e2e/apimachinery/webhook.go]: Those sound more like integration-level tests, exercising all the permutations of responses. I wouldn't really expect this test to expand to do all of those.

**Comment by hh** [comment]
We will need to wait until AdmissionWebhook is GA BEFORE we promote test to conformance.
/close

**Pull request closed without merging**